### DISCUSSION TITLE
Please tell me your homecountry!　あなたの出身地を教えてください！

### DISCUSSION THREAD

**Discussion by yuito1227** [NAIST-SE/AttractingContributors#535] [General]
Please tell me your home country!

あなたの出身地を教えてください！

https://github.com/yuito1227/tell-me-your-homecountry.git

**Comment by ani-hovhannisyan** [comment]
Hi, I have added # 12 in your project, can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
